cvdupdate/auto_updater.py

```diff
@@ -2,21 +2,23 @@
 
 from cvdupdate.cvdupdate import CVDUpdate
 
-# one time per day
-WAIT_TIME_SECONDS = 24 * 60 * 60
+def start(interval: int) -> None:
+    '''Spawns a thread to update the AV db after "interval" seconds
```

Comment:
- OCD: Please use `"""` and not `'''`
- OCD again: `Spawns` -> `Spawn` https://stackoverflow.com/a/59902236